### ISSUE TITLE
(Security) Local File Disclosure Vulnerability on "/demo/static/" embed functionality

### ISSUE THREAD

**Issue by samwcyo** [mapproxy/mapproxy#526]
## Summary
There is a local file disclosure vulnerability in the mapproxy demo site. This would allow an attacker to read arbitrary files.

## Steps to Reproduce
1.) Send the following HTTP request to a box running mapproxy:

```
GET /demo/static/../../../../../../../../../../etc/passwd HTTP/1.1
```

2.) Observe the `/etc/passwd` in the HTTP response

**Comment by olt** [comment]
Thanks a lot for reporting this issue.
This is embarrassing and an example how things become a problem when underlying assumptions change. 11 years ago, packages where commonly installed inside zip files (eggs) and pkg_resource handled only file access inside these zip files.

Disabling the demo service is the quickest solution. Will push a fix for the demo service now as well.
 
cc @mapproxy/psc

**Comment by weskamm** [comment]
should be fixed with released version 1.13.2
